package.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "terriajs",
-  "version": "8.7.10",
+  "version": "8.7.11",
   "description": "Geospatial data visualization platform.",
   "license": "Apache-2.0",
   "engines": {
     "node": ">= 16.0.0"
   },
-  "sideEffects": false,
```

Comment:
This is apparently important for tree shaking. There is a granular version of sideEffects where you provide paths, and I think that would make sense since there are so few imports with side effects in the tree currently. Long-term, I'm guessing you want to get rid of side effecting imports.

Reply:
It is not always easy to properly setup the list of files, and I don't want to block the webpack upgrade with that

Reply:
True, it's already been sitting for 2 months, would be nice to get these big PRs into main.

Reply:
Yeah, we should remove this setting. I was mostly testing to see if it broke anything - it doesn't seem to. I think we can revisit this later.